db/warehouse/migrate/20230828180743_copy_over_vamc_column.rb

```diff
@@ -0,0 +1,5 @@
+class CopyOverVamcColumn < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
This is probably fine, but it should probably be:
```suggestion
  def up
```